_includes/footer.html

```diff
@@ -7,9 +7,9 @@
   <p>Copyright 2009-2015 Software Freedom Conservancy, Sugar Labs®</p>
 
   <!-- FIXME: Update/remove pages -->
-  <p>Connect with us on
-    <a href="https://socialhelp.sugarlabs.org">Social Help</a> or
-    <a href="https://www.facebook.com/SugarLabs-187845102582/timeline/">Facebook</a> or
-    <a href="https://github.com/sugarlabs">Github</a>
-  </p>
+  <h1>
+    <a href="https://www.facebook.com/SugarLabs-187845102582/timeline/" target="_blank" title="Facebook"><i class="fa fa-facebook"></i></a>
+    <a href="https://github.com/sugarlabs" target="_blank" title="Github"><i class="fa fa-github"></i></a>
+    <a href="https://socialhelp.sugarlabs.org/" target="_blank" title="Social Help"><i class="fa fa-question-circle"></i></a>
```

Comment:
Please use the same icon that is used in sugar.  Users need consistency because otherwise they get confused.